spec/support/fake_objects.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'cucumber/core/filter'
+
+# The following fake objects was previously declared within `describe` scope.
+# Declaring into scope did not isolate them.
+#
+# Moving those into a dedicated support file explitely state that those are available
```

Comment:
`# Moving these into a dedicated support file explicitly states that these are available globally as soon as they are required once.`